Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/ListBlobsInContainer.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.ComponentModel;
+using Azure.Storage.Blobs;
+using Azure;
+using Frends.AzureBlobStorage.ListBlobsInContainer.Definitions;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Models;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Frends.AzureBlobStorage.ListBlobsInContainer
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// List blobs and subdirectories in Azure Storage container with flat or hierarchical listing structure.
+        /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.AzureBlobStorage.ListBlobsInContainer)
+        /// </summary>
+        /// <param name="source">Source connection parameters.</param>
+        /// <param name="optional">Optional parameters</param>
+        /// <returns>object { string BlobType, string Uri, string Name, string ETag }</returns>
```

Comment:
Please update BlobData properties here. You are missing `ListingStructure` and `Type`. There is no property named `BlobType`.